packages/mc-html-template/lib/load-env.js

```diff
@@ -1,16 +1,53 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
 const fs = require('fs');
 
 // Keep a reference to the loaded config so that requiring the module
 // again will result in returning the cached value.
 let loadedEnv;
 
-module.exports = configPath => {
-  if (loadedEnv) return loadedEnv;
+const START_OF_ENV_PLACEHODER = 'env:';
```

Comment:
```suggestion
const START_OF_ENV_PLACEHOLDER = 'env:';
```

Reply:
Any reason not to use the syntax `${env:MY_ENV}`?

Reply:
Not really. I timeboxed 30 minutes for this that's why kind of :)